src/localize/localize.ts

```diff
@@ -44,6 +45,7 @@ const languages: any = {
   fr,
   he,
   hu,
+  is,
```

Comment:
thanks for the PR. can you please add `is` to the list of supported languages? thanks https://github.com/pkissling/clock-weather-card/blob/master/README.md?plain=1#L145

Reply:
done